tap_twitter_ads/schemas/cards.json

```diff
@@ -0,0 +1,44 @@
+{
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+      "name": {
+        "type": ["null", "string"]
+      },
+      "components": {
+        "anyOf": [
+          {
+            "type": "array",
+            "items": {
+              "type": ["null", "object"],
+              "properties": {}
+            }
+          },
+          {
+            "type": "null"
+          }
+        ]
```

Comment:
We don't need to define `null` separately inside `anyof`, instead we can just do:
```

"type": ["null", "array"],
"items": {
              "type": ["null", "object"],
              "properties": {}
            }